CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+## [v0.62.0-rc.0.0]
+
+### Breaking
+
+- Removing <TextInput>'s `onTextInput` event ([3f7e0a2](https://github.com/facebook/react-native/commit/3f7e0a2) by [@shergin](https://github.com/shergin))
+- Restore behavior for `underlayColor={null}` in `TouchableHighlight`. ([37d8440](https://github.com/facebook/react-native/commit/37d8440) by [@yungsters](https://github.com/yungsters))
+
+### Added
+
+- Remove requestAnimationFrame when focusing input on mount ([5798cf2](https://github.com/facebook/react-native/commit/5798cf2) by [@janicduplessis](https://github.com/janicduplessis))
+- Added missing `console` polyfills in release builds. ([b7ab922](https://github.com/facebook/react-native/commit/b7ab922) by [@yungsters](https://github.com/yungsters))
+- Add missing accessibility props on Touchables ([8c0c860](https://github.com/facebook/react-native/commit/8c0c860) by [@xuelgong](https://github.com/xuelgong))
+- Add getNativeScrollRef method to FlatList component ([bde1d63](https://github.com/facebook/react-native/commit/bde1d63))
+- Clean up unused functions on ReactContext ([3963c7a](https://github.com/facebook/react-native/commit/3963c7a) by [@ejanzer](https://github.com/ejanzer))
```

Comment:
```suggestion
```
I believe this should be an internal change; the removed code was already flagged as "do not use".